src/ogc/wcs/WcsCapabilities.js

```diff
@@ -61,6 +61,20 @@ define([
             this.assembleDocument();
         };
 
+        /**
+         * Returns the GetCoverage base url as detailed in the capabilities document
+         * @param coverageId the coverage id or name
```

Comment:
Is this comment obsolete, or is getCoverageBaseUrl missing its coverageId argument?

Reply:
Obsolete, actually, never actually needed.